pkg/volume/storageos/storageos_util.go

```diff
@@ -76,13 +78,17 @@ type apiImplementer interface {
 
 // storageosUtil is the utility structure to interact with the StorageOS API.
 type storageosUtil struct {
-	api apiImplementer
+	api  apiImplementer
+	host volume.VolumeHost
 }
 
 func (u *storageosUtil) NewAPI(apiCfg *storageosAPIConfig) error {
 	if u.api != nil {
 		return nil
 	}
+	if u.host == nil {
+		return nil
```

Comment:
returning nil here changes the behavior of anything currently using storageosUtil... I would expect this to error or continue with unfiltered dials